axum/src/extract/multipart.rs

```diff
@@ -50,6 +50,13 @@ use std::{
 /// let app = Router::new().route("/upload", post(upload));
 /// # let _: Router = app;
 /// ```
+/// 
+/// # Large Files
+/// 
+/// For security reasons, by default, all axum request bodies (including multipart uploads) are limited to 2MB. 
+/// To upload files larger than this, use [DefaultBodyLimit][default-body-limit].
+/// 
+/// [default-body-limit]: crate::extract::DefaultBodyLimit
```

Comment:
I think I'd rephrase this slightly. "all axum request bodies" isn't technically true since extractors themselves have to enforce the limit. For example extracting `Request<Body>` and then consuming the body wont consider the default limit. You have to do that yourself.

Multipart enforces the limit [here](https://github.com/tokio-rs/axum/blob/main/axum-extra/src/extract/multipart.rs#L102-L105)

```suggestion
/// # Large Files
/// 
/// For security reasons, by default, `Multipart` limits the request body size to 2MB. 
/// See [`DefaultBodyLimit`][default-body-limit] for how to configure this limit.
/// 
/// [default-body-limit]: crate::extract::DefaultBodyLimit
```